contracts/Certificate.sol

```diff
@@ -8,19 +8,20 @@ import "@openzeppelin/contracts-upgradeable/token/ERC777/ERC777Upgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC777/IERC777RecipientUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/utils/introspection/ERC1820ImplementerUpgradeable.sol";
 import "./ERC1155PresetPausableNonTransferrable.sol";
+import {MintDisabled} from "./SharedCustomErrors.sol";
```

Comment:
if we're going to share this error, I think a generic `disabled function` error is better-- you can infer which function was disabled by the call itself